Cargo.toml

```diff
@@ -45,7 +45,7 @@ features = [
     "kad",
     "mdns",
     "ping",
-    "relay",
+    #"relay",
```

Comment:
So no relay?

Reply:
I was playing around with it. It's not really that useful if you are using quic. Also it's extremely slow since you need to encrypt and multiplex everything twice (as the relay opens a new encrypted/multiplexed connection over a substream to the relay)

Reply:
by not being that useful I mean it doesn't currently work.